packages/admin-portal/src/actions/approvals.ts

```diff
@@ -1,9 +1,11 @@
 import { actionCreator } from '../utils/actions'
 import ActionTypes from '../constants/action-types'
-import { ApprovalsList } from '@/reducers/approvals'
+import { PagedResultApprovalModel_ } from '@reapit/foundations-ts-definitions'
 
 export const approvalsRequestData = actionCreator<number>(ActionTypes.APPROVALS_REQUEST_DATA)
 export const approvalsRequestDataFailure = actionCreator<void>(ActionTypes.APPROVALS_REQUEST_FAILURE)
 export const approvalsLoading = actionCreator<boolean>(ActionTypes.APPROVALS_LOADING)
-export const approvalsReceiveData = actionCreator<ApprovalsList | undefined>(ActionTypes.APPROVALS_RECEIVE_DATA)
+export const approvalsReceiveData = actionCreator<PagedResultApprovalModel_ | undefined>(
```

Comment:
I think undefined can be remove

Reply:
Please update the actions name and constant using our new convention:
approvalsRequestData => fetchApprovalList
approvalsRequestDataFailure => fetchApprovalListFailed
approvalsReceiveData => fetchApprovalListSuccess

Also, I think `approvalsLoading` should be removed because loading state can be hanlded by 3 actions above. No need to create a single action for it,

Reply:
@nphivu414 noted, will do but not in this PR, as I'm working on refactor the state tree